hack/ci-deploy-release.sh

```diff
@@ -78,8 +78,13 @@ else
  kubectl_create -f="${SO_NODECONFIG_PATH}"
 fi
 
-kubectl_create -n=local-csi-driver -f="${source_url}/${revision}/examples/common/local-volume-provisioner/local-csi-driver/"{00_namespace.yaml,00_scylladb-local-xfs.storageclass.yaml,10_csidriver.yaml,10_driver.serviceaccount.yaml,10_provisioner_clusterrole.yaml,20_provisioner_clusterrolebinding.yaml,50_daemonset.yaml}
-kubectl -n=local-csi-driver rollout status --timeout=5m daemonset.apps/local-csi-driver
+SO_CSI_DRIVER_PATH="${SO_CSI_DRIVER_PATH=${source_url}/${revision}/examples/common/local-volume-provisioner/local-csi-driver/}"
```

Comment:
URLs can't list directories

Reply:
fixed, although with a use of `eval`. Let me know if this is fine

Reply:
> even 2 kubectl apply statements are fine

Done, I couldn't find a different way to do this (other than eval).